src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/DelayLoadHelperImport.cs

```diff
@@ -29,6 +32,7 @@ public class DelayLoadHelperImport : Import
             string callSite = null)
             : base(importSectionNode, instanceSignature, callSite)
         {
+            Target = factory.Target;
```

Comment:
This doesn't look necessary - NodeFactory is passed as the `factory` argument to `EncodeData`.

Reply:
updated